packages/react-router/modules/Redirect.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { createLocation, locationsAreEqual } from "history";
```

Comment:
This is still used below, so you need to leave it. There isn't test coverage, but the else block will fail otherwise.

Reply:
Remnant of my chain of experimentation. How did that not cause a build failture?

Reply:
Re-added. And the new unit test should cover that now.